[2025-01-24T07:34:30.109+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T07:34:30.115+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task_yellow_tripdata_2021-02.csv manual__2025-01-24T07:33:58.318067+00:00 [queued]>
[2025-01-24T07:34:30.117+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task_yellow_tripdata_2021-02.csv manual__2025-01-24T07:33:58.318067+00:00 [queued]>
[2025-01-24T07:34:30.117+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-24T07:34:30.121+0000] {taskinstance.py:2889} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task_yellow_tripdata_2021-02.csv> on 2025-01-24 07:33:58.318067+00:00
[2025-01-24T07:34:30.124+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=724) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T07:34:30.124+0000] {standard_task_runner.py:72} INFO - Started process 725 to run task
[2025-01-24T07:34:30.125+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'bigquery_external_table_task_yellow_tripdata_2021-02.csv', 'manual__2025-01-24T07:33:58.318067+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/GCP Project/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp6mr5hzx4']
[2025-01-24T07:34:30.126+0000] {standard_task_runner.py:105} INFO - Job 84: Subtask bigquery_external_table_task_yellow_tripdata_2021-02.csv
[2025-01-24T07:34:30.140+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task_yellow_tripdata_2021-02.csv manual__2025-01-24T07:33:58.318067+00:00 [running]> on host 22ff6bf154fe
[2025-01-24T07:34:30.165+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='bigquery_external_table_task_yellow_tripdata_2021-02.csv' AIRFLOW_CTX_EXECUTION_DATE='2025-01-24T07:33:58.318067+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-24T07:33:58.318067+00:00'
[2025-01-24T07:34:30.165+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T07:34:30.173+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-01-24T07:34:30.173+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-01-24T07:34:30.173+0000] {credentials_provider.py:402} INFO - Getting connection using `google.auth.default()` since no explicit credentials are provided.
[2025-01-24T07:34:32.323+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T07:34:32.324+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=bigquery_external_table_task_yellow_tripdata_2021-02.csv, run_id=manual__2025-01-24T07:33:58.318067+00:00, execution_date=20250124T073358, start_date=20250124T073430, end_date=20250124T073432
[2025-01-24T07:34:32.360+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T07:34:32.371+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-24T07:34:32.371+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
